src/Components/NavBar/menuData.ts

```diff
@@ -37,70 +37,25 @@ export const ARTISTS_SUBMENU_DATA: MenuLinkData = {
       },
       {
         text: "Trending Now",
-        href: "/collection/trending-this-week",
-      },
-      {
-        text: "Artsy Vanguard Artists",
-        href: "/collection/the-artsy-vanguard",
-        dividerBelow: true,
+        href: "/collection/trending-this-week/new-this-week",
       },
       {
         text: "Curator's Picks",
         menu: {
           title: "Curator's Picks",
           links: [
             {
-              text: "Rising Artists of Africa & Its Diaspora",
-              href: "/collection/rising-artists-of-africa-and-its-diaspora",
-            },
-            {
-              text: "Rising Artists of Asia & Its Diaspora",
-              href: "/collection/rising-artists-of-asia-and-its-diaspora",
+              text: "Curators’ Picks: Emerging",
+              href: "/collection/curators-picks-emerging",
             },
             {
-              text: "Women Artists Now",
-              href: "/collection/women-artists-to-watch",
+              text: "Curators’ Picks: Blue-Chip",
```

Comment:
is it like this or `Curator's`?
Line 43 is in this form and here it's in a different form

Reply:
Good question 🤔  
It's like that on the ticket, but maybe @jpoczik can confirm?
It's a bit odd that the main header would refer to an individual curator, but the sub links to the collective. But, I'm not a native speaker 🤷

Reply:
I believe it should match what you see when you go to the page:

https://www.artsy.net/collection/curators-picks-blue-chip?metric=in

Which looks like this:

```
Curators’ Picks: Blue-Chip
```

Reply:
yes confirming what jon says that it's from multiple curators so **Curators’ Picks: Blue-Chip** is correct